docs/admin/kubeadm-init.md

```diff
@@ -0,0 +1,425 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm init
+notitle: true
+---
+{% capture overview %}
+## Kubeadm init {#cmd-init}
+{% endcapture %}
+
+{% capture body %}
+{% include_relative _kubeadm/kubeadm_init.md %}
+
+### Init workflow {#init-workflow}
+`kubeadm init` bootstraps a Kubernetes master node by executing the
+following steps:
+
```

Comment:
In another comment, I suggested capitalizing Kubeadm when it is at the beginning of a sentence. But in cases like this, I agree with what you have done. Lower case for a specific command, even if it is at the beginning of a sentence.